validation/genesis/validation-inputs/183/meta.toml

```diff
@@ -0,0 +1,4 @@
+genesis_creation_commit = "9047beb54c66a5c572784efec8984f259302ec92"
+node_version = "18.12.1"
+monorepo_build_command = "pnpm"
+genesis_creation_command = "opnode2"
```

Comment:
Please change this from `opnode2` to `forge1`. This matches the command used for the Ethernity testnet and should get the `just validate-genesis-allocs 183` ci test to pass

Reply:
Updated, `just validate-genesis-allocs 183` passed locally